docs/deploying/azure.md

```diff
@@ -0,0 +1,81 @@
+---
+permalink: /docs/deploying/azure/index.html
+layout: docs
+---
+
+If you've been following along with [Getting Started](../README.md), it's time to deploy so you can use it beyond just your local machine.
+[Azure](http://http://azure.microsoft.com/) is a somewhat easy and probably unsupported way to deploy hubot.
+
+You will need to install the azure-cli via npm after you have follow the initial instructions for your hubot.
+
+    % npm install -g azure-cli
+
+Inside your new hubot directory, make sure you've created a git repository, and that your work is committed:
+
+    % git init
+    % git add .
+    % git commit -m "Initial commit"
+
+Then create a GitHub repository for your hubot. This is where zzure will pull your code from instead of needing to deploy directly from your dev machine to Azure.
```

Comment:
Could link 'create a GitHub repository' to https://help.github.com/articles/create-a-repo/